### DISCUSSION TITLE
Welcome to Wolfram Language Server Discussions!

### DISCUSSION THREAD

**Discussion by kenkangxgwe** [kenkangxgwe/lsp-wl#69] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by topazus** [comment]
Hello, I'm a newcomer to Wolfram language learning, so there are not so many things that I can help in the community. I think that Wolfram language is pretty useful in matrix manipulation, equation solving, integral calculation etc, which are the learning contents that I recently found it very interesting to study.
